frigate/config.py

```diff
@@ -795,6 +814,11 @@ def verify_config_roles(camera_config: CameraConfig) -> None:
             f"Camera {camera_config.name} has rtmp enabled, but rtmp is not assigned to an input."
         )
 
+    if camera_config.audio.enabled and "audio" not in assigned_roles:
+        raise ValueError(
+            f"Camera {camera_config.name} has audio events enabled, but audio is not assigned to an input."
+        )
```

Comment:
It feels like one of this settings is superfluous and unnecessary

Reply:
`audio` must be assigned to a specific role, we can not assume which role it is assigned to. This is in line with how the other roles in frigate works.

Reply:
other roles doesn't requires second setting at camera level. for example, if you add "record" role to input, you doesn't need to additionaly enable recording in camera level

Reply:
> for example, if you add "record" role to input, you doesn't need to additionaly enable recording at camera level

that is absolutely incorrect https://github.com/blakeblackshear/frigate/pull/6444

record must be enabled in the config and added as a role to be turned on

Reply:
Oh. My apologies